private/bufpkg/bufanalysis/bufanalysistesting/bufanalysistesting_test.go

```diff
@@ -24,43 +25,52 @@ import (
 
 func TestBasic(t *testing.T) {
 	t.Parallel()
-	fileAnnotation := newFileAnnotation(
+	fileAnnotations := []bufanalysis.FileAnnotation{
+		newFileAnnotation(
+			t,
+			"path/to/file.proto",
+			1,
+			0,
+			1,
+			0,
+			"FOO",
+			"Hello.",
+		),
+		newFileAnnotation(
+			t,
+			"path/to/file.proto",
+			2,
+			1,
+			2,
+			1,
+			"FOO",
+			"Hello.",
+		),
+	}
+	sb := &strings.Builder{}
+	err := bufanalysis.PrintFileAnnotations(sb, fileAnnotations, "text")
+	require.NoError(t, err)
+	assert.Equal(
 		t,
-		"path/to/file.proto",
-		1,
-		0,
-		1,
-		0,
-		"FOO",
-		"Hello.",
+		`path/to/file.proto:1:1:Hello.`+"\n"+
```

Comment:
You don't need to do this, you can just do:

```
`This
is how you do multiline strings
and more`
```

Reply:
Tried that, gofmt is formatting it like this,
```go
	assert.Equal(
		t,
		`path/to/file.proto:1:1:Hello.
path/to/file.proto:2:1:Hello.
`,
		sb.String(),
	)
```

Reply:
Yep thats right